app/controllers/vehicles_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   end
 
   def create
-    @vehicle = Vehicle.create(vehicle_params)
+    @vehicle = CreateVehicle.create(vehicle_params)
```

Comment:
I understand you, but I don't like the idea to put the some logic that changes of object in the controller. But show me, how can I do this or  another way! 😛